pom.xml

```diff
@@ -1106,15 +1106,39 @@
         <scope>${flume.deps.scope}</scope>
         <exclusions>
           <exclusion>
+            <groupId>commons-codec</groupId>
```

Comment:
I don't really disagree with this, since there is only one usage of flume in the project now. But the exclusions are specific to the `flume-assembly` module's needs, not to consumers of Flume in general in the project right? can this be managed down in the module alongside the other changes for the same purpose?